packages/retail-ui-extensions-react/src/context.ts

```diff
@@ -2,7 +2,7 @@ import {createContext} from 'react';
 import {
   ApiForRenderExtension,
   RenderExtensionPoint,
-} from '@shopify/retail-ui-extensions/src';
+} from '@shopify/retail-ui-extensions';
```

Comment:
Is it going to fix [this issue](https://github.com/Shopify/pos-next-react-native/issues/13732)